pmpro-network.php

```diff
@@ -96,8 +106,8 @@ function pmpron_pmpro_checkout_boxes()
 	<div id="pmpro_site_fields" class="pmpro_checkout">
 		<hr />
 		<h3>
-			<span class="pmpro_checkout-h3-name"><?php _e( 'Site Information', 'pmpro-network'); ?></span>
-			<span class="pmpro_checkout-h3-msg"><?php echo sprintf( 'Sites Included: <strong>%s</strong>', pmpron_getSiteCredits($level_id), 'pmpro-network'); ?></span>
+			<span class="pmpro_checkout-h3-name"><?php esc_html_e( 'Site Information', 'pmpro-network'); ?></span>
+			<span class="pmpro_checkout-h3-msg"><?php echo sprintf( __( 'Sites Included: <strong>%s</strong>', 'pmpro-network' ), pmpron_getSiteCredits($level_id), 'pmpro-network'); ?></span>

```

Comment:
This tweak is tricky because we are using HTML in the translated text, so `esc_html()` won't do what we hope here without addressing that aspect first

Reply:
Ah right I missed the strong tags. Can probably just scrap this change for now and deal with it later?